src/Realm.C

```diff
@@ -2716,13 +2716,7 @@ Realm::register_nodal_fields(stk::mesh::Part* part)
   // register high level common fields
   // Declare volume/area_vector fields
   const int numVolStates = does_mesh_move() ? number_of_states() : 1;
```

Comment:
@gantech @ashesh2512 is this right? Shouldn't it be `has_mesh_deformation()`?  We only need multiple dnv states for deformation right?